src/lib/lightnet.js

```diff
@@ -0,0 +1,703 @@
+      }
+      await fs.remove(lightnetConfigFile);
+    });
+  }
+  if (logsDir) {
+    const boldLogs = chalk.reset.bold('logs');
+    console.info(
+      `\nThe Docker container processes ${boldLogs} can be found at:\n\n` +
+        chalk.green.bold(logsDir) +
+        '\n'
+    );
+    console.info('Done\n');
+  } else {
+    if (
+      fs.existsSync(lightnetLogsDir) &&
+      fs.readdirSync(lightnetLogsDir).length === 0
+    ) {
+      fs.removeSync(lightnetLogsDir);
+    }
+    console.info('\nDone\n');
+  }
+}
+
+/**
+ * Gets the lightweight Mina blockchain network status.
+ * @param {boolean} object.preventDockerEngineAvailabilityCheck - Whether to prevent the Docker Engine availability check.
+ * @returns {Promise<void>}
+ */
+export async function lightnetStatus({
+  preventDockerEngineAvailabilityCheck = false,
+} = {}) {
+  if (!preventDockerEngineAvailabilityCheck) {
+    await checkDockerEngineAvailability();
+  }
+  const containerStatus = getDockerContainerStatus(lightnetDockerContainerName);
+  if (containerStatus === 'not found') {
+    console.log(
+      chalk.red(
+        '\nThe lightweight Mina blockchain network Docker container does not exist!'
+      )
+    );
+    shell.exit(1);
+  }
+  console.log('\n' + chalk.reset.bold('Lightweight Mina Blockchain Network'));
+  if (containerStatus === 'running' && fs.existsSync(lightnetConfigFile)) {
+    printUsefulUrls();
+    printDockerContainerProcessesLogPaths();
+    await printBlockchainNetworkProperties();
+    printZkAppSnippet();
+    printExtendedDockerContainerState(lightnetDockerContainerName);
+  } else {
+    console.info(
+      chalk.yellow.bold(
+        '\nWarning:\nThe lightweight Mina blockchain network Docker container is either ' +
+          '\nnot running or it was created outside of this application.' +
+          '\nOnly limited information is available.'
+      )
+    );
+    printExtendedDockerContainerState(lightnetDockerContainerName);
+  }
+}
+
+async function checkDockerEngineAvailability() {
+  await step('Checking Docker Engine availability', async () => {
+    if (!shell.which('docker')) {
+      console.info(
+        '\n\nPlease ensure that Docker Engine is installed, then try again.' +
+          '\nSee https://docs.docker.com/engine/install/ for more information.'
+      );
+      shell.exit(1);
+    }
+    const { code } = shell.exec('docker ps -a', { silent: true });
+    if (code !== 0) {
+      console.info(
+        '\n\nPlease ensure that Docker Engine is running, then try again.'
+      );
+      shell.exit(1);
+    }
+  });
+}
+
+async function handleStartCommandChecks(containerName) {
+  if (
+    getDockerContainerStatus(containerName) === 'running' &&
+    fs.existsSync(lightnetConfigFile)
+  ) {
+    console.log(
+      chalk.red(
+        '\n\nThe lightweight Mina blockchain network is already running!'
+      )
+    );
+    shell.exit(1);
+  } else if (
+    getDockerContainerStatus(containerName) !== 'not found' &&
```

Comment:
Could we refactor these string error codes into an object that has a consistent mapping? Otherwise, in the case of refactoring it'll be less of a pain.

Something like:
```js
const ErrorMessages = {
    CONTAINER_NOT_FOUND: "not-found"
    ...
}
```